bpf/lib/nodeport.h

```diff
@@ -423,7 +424,7 @@ static __always_inline int dsr_reply_icmp6(struct __ctx_buff *ctx,
 {
 #ifdef ENABLE_DSR_ICMP_ERRORS
 	const __s32 orig_dgram = 64, off = ETH_HLEN;
-	const __u32 l3_max = sizeof(*ip6) + orig_dgram;
+       __u8 tmp[orig_dgram];
```

Comment:
nit: Could you find a better naming than `tmp` for this buffer storing original ip hdr + proto?

Reply:
done